### DISCUSSION TITLE
Why are apostrophes changed to  \' on my web server?

### DISCUSSION THREAD

**Discussion by ghost** [datenstrom/community#142] [Ask a question] [answered]
Hello,

When I try to edit a page and write an apostrophe like in **Let's talk**, a backslash appears, like this:

> Let\'s talk

When I delete the backslash, it reappears.

When I backslash the backslash, I get "something went wrong on the server."

I looked in the Issues archive, and can't find anything.

Perhaps it's my configuration, the server, the file format, or Yellow. I suspect it's my configuration, since when I add "[yellow]" to my page, nothing happens and "[yellow]" appears.

Where should I look?

Many thanks!

**Comment by markseuffert** [comment, marked as answer]
Can you please update to latest version of Yellow? It looks like a problem with PHP's [magic quotes](https://en.wikipedia.org/wiki/Magic_quotes), which the latest version should take care of. Let me know if the update fixes the problem.

**Comment by ghost** [comment]
I've updated the whole package, and I now can edit my page. Thanks!

However, it seems that yellow.php lost contact with the configuration file. (My website name is now "Yellow," contrary to what I've written in config.ini.) This does not seem to be a problem covered so far in your list of issues. Should I open another thread for that?

**Comment by markseuffert** [comment]
Oh that's my fault, I should have said that the configuration file format has changed.

Here's how to update: Go to the command line and run `php yellow.php version`. This will update the configuration files from an older version of Yellow and your website name should be back.

Sorry for any troubles and thanks for the feedback. Please let me know if this works for you.

**Comment by ghost** [comment]
I don't think I can do that, since it's only a rented space. Can do I do this locally and then upload the files, and if so, how?

**Comment by markseuffert** [comment]
With Mac or Linux you can do it on your own computer. Otherwise contact me and I'll fix it for you.

**Comment by ghost** [comment]
Email sent. Meanwhile, I'll see if I can't find a way to run PHP from a USB key. Creating static sites might be good for security reasons.

**Comment by ghost** [comment]
It works! Thanks!
